corehq/apps/hqadmin/tasks.py

```diff
@@ -142,7 +139,7 @@ def send_abnormal_usage_alert(alert):
     )
     send_html_email_async(
         subject,
-        support_email,
+        settings.SUPPORT_EMAIL,
```

Comment:
what would happen if there is no email set in settings?

Reply:
Hmm, I don't know. There are default values for all of the emails in settings, so I'm not worried about it.

Reply:
Sounds good, thanks for confirming